dht/boot_strapper.hh

```diff
@@ -52,7 +53,7 @@ public:
         , _token_metadata_ptr(std::move(tmptr)) {
     }
 
-    future<> bootstrap(streaming::stream_reason reason, gms::gossiper& gossiper, inet_address replace_address = {});
+    future<> bootstrap(streaming::stream_reason reason, gms::gossiper& gossiper, service::frozen_topology_guard, inet_address replace_address = {});
```

Comment:
Why local function gets frozen guard which suppose to be "in transit" representation?

Reply:
Answered here: https://github.com/scylladb/scylladb/pull/15847#discussion_r1384195904